docs/docs/manual/grelfunctions.md

```diff
@@ -174,9 +174,9 @@ Returns a string converted to a number. Will attempt to convert other formats in
 
 Returns the array of strings obtained by splitting s by sep. The separator can be either a string or a regex pattern. For example, `"fire, water, earth, air".split(",")` returns an array of 4 strings: [ "fire", " water", " earth", " air" ]. Note that the space characters are retained but the separator is removed. If you include “true” for the preserveTokens boolean, empty segments are preserved.
 
-###### splitByLengths(s, n1, n2, ...)
+###### splitByLengths(s, n1, n2, ...)[i1,i2] 
 
-Returns the array of strings obtained by splitting s into substrings with the given lengths. For example, `"internationalization".splitByLengths(5, 6, 3)` returns an array of 3 strings: [ "inter", "nation", "ali" ]. Excess characters are discarded.
+Returns the array of strings, between i1 and (i2 - 1), obtained by splitting s into substrings with the given lengths. For example, `"internationalization".splitByLengths(5, 6, 3)[0,2]` returns an array of 2 strings: [ "inter", "nation" ]. "ali" is desgraded as it falls outside of the indices. Excess characters are also disregarded.
```

Comment:
I think the original misunderstanding reported in #3929 was that the user was expecting something like:

"University of Washington".splitByLengths(4)

to return
["Univ","ersity of Washington"]

but it actually returns
["Univ"]

I think what is needed here is just to make this even more explicit - so expanding on the existing comment "Excess characters are discarded" to make it extra clear - so taking what you have already written something like:

"The remaining characters (in this case "zation") are discarded and not included in the array output"